tasks/src/video-classification/about.md

```diff
@@ -0,0 +1,69 @@
+## Use Cases
+Video classification models can be used to categorize what a video is all about.
+
+### Activity Recognition
+Video classification models are used to perform activity recognition which are useful for fitness applications. Activity 
+recognition is also helpful for vision-impaired individuals especially when they're commuting.
+
+### Video Search
+Models trained in video classification can improve user experience by organizing and categorizing video galleries on the phone or in the cloud, on multiple keywords or tags.
+
+## Inference
+
+Below you can find code for inferring with a pre-trained video classification model.
+
+```python
```

Comment:
This definitely needs more comments as we would like to also cover a beginner audience if this sounds good 🙂

Reply:
I meant, can you add more explanation on why we need stuff like `VideoReader`